### DISCUSSION TITLE
Multiple roles for contributors

### DISCUSSION THREAD

**Discussion by rkrug** [eblondel/zen4R#87] [Q&A] [answered]
I have a case, where each contributor has multiple roles. Do I really have to specify each contributors multiple times, for each contribution once?

**Comment by eblondel** [comment, marked as answer]
That's a good question that we could forward to Zenodo team. At that time, the metadata model plans that you can add one role per person, not various. Unless you specify as you say a person multiple times, you would be constrained to put one role by person. This is probably a constraint inherited from the Datacite schema, that we see in other metadata standards like ISO 19115.

**Reply by rkrug** [reply]
So I have either to decide I=on one role, or add each contributor multiple times.

Thanks a lot.

**Reply by eblondel** [reply]
yes, unfortunately. the best would be to have a "main" role by person, but that's not always the case. I have the same problem with producing ISO 19115 where people have various roles, and we have to duplicate multiple contact blocks just because of the role that changes
